### ISSUE TITLE
[META] Site "Status report" page issues.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#283]
I've been testing Backdrop more and more lately and I'm doing so on a local nginx/MySQL/PHP stack on my Windows laptop. I assume that a lot of people that wish to give Backdrop a spin will be doing so locally too before deploying on actual hosting environments.

Now I'm a "all green" maniac when it comes to the site status page. From my experience I know that it's ok for some things to not be ok in this page, but from a newcomer perspective it is scary to see too many errors. Especially when they don't matter, don't make sense, do not apply in certain cases or if no action can be taken on specific scenarios.

So, here's a list of warnings in the "Status report" page that either cannot be resolved on specific cases or don't make sense at all:
- [x] https://github.com/backdrop/backdrop-issues/issues/396 and https://github.com/backdrop-ops/backdropcms.org-issues/issues/22 | Backdrop CMS update status: Up to date, "There was a problem checking available updates for Backdrop.": Until we actually have an update server in place this should temporarily be disabled (gray or blue instead of red).
- [x] https://github.com/backdrop/backdrop-issues/issues/395 | Configuration file: Not protected. The `settings.php` file can be set to read-only _on Windows environments_, but the backdrop folder cannot be (unless you set _every_ folder and file within it to read-only too - and we don't want that). So, again, _on Windows_ we should "nag" until the `setting.php` is set to read-only, but once that is done we should make the warning gray/blue.
- [x] https://github.com/backdrop/backdrop-issues/issues/472 | Upload progress: Not enabled: this one is already blue, but the message says "Your server is not capable of displaying file upload progress. File upload progress requires an Apache server running PHP with mod_php.". Now, in the case of nginx on Windows at least, I do have the php_uploadprogress extension enabled (I verified through php info), so this message doesn't work as it should and doesn't make sense. It does not help the user and causes confusion. As a matter of fact there is this issue in the Drupal queue for D8: [Make core aware of Nginx and PHP-FPM to avoid confusing alerts](https://www.drupal.org/node/1559116) and there's also [Add support of PHP 5.4 build-in uploadprogress](https://www.drupal.org/node/1561866)

Any other OS-specific use cases that are related to the "Status report" page messages, please add them here as tasks and lets discuss what would be the reasonable thing to do. I guess separate issues should be filed for each case.

**Comment by quicksketch** [comment]
With 1-3 addressed. I feel like we should close this issue. The redesign of the status report page is a pretty big unknown, as we don't have anything to go on other than it should be prettier.

**Comment by jenlampton** [comment]
I agree.  We have another issue for the redesign of the status report so we won't loose track of that.
